db/migrate/20170110090935_add_ems_to_miq_alert_status.rb

```diff
@@ -0,0 +1,20 @@
+class AddEmsToMiqAlertStatus < ActiveRecord::Migration[5.0]
+  class MiqAlertStatus < ActiveRecord::Base; end
+
+  def up
+    add_column :miq_alert_statuses, :ems_id, :bigint
+    say_with_time("add ems_id to miq alert statuses") do
+      %w(vms hosts ems_clusters container_images).each do |alert_resource|
+        arel_table = Arel::Table.new(alert_resource.to_sym)
+        klass_name = alert_resource.classify
```

Comment:
Think `klass_name` for `vms` needs to be `VmOrTemplate` - just add a followup `if` clause